packages/auth-client/src/index.ts

```diff
@@ -219,7 +239,11 @@ export class AuthClient {
       if (maybeIdentityStorage) {
         try {
           if (typeof maybeIdentityStorage === 'object') {
-            key = await ECDSAKeyIdentity.fromKeyPair(maybeIdentityStorage);
+            if (keyType === 'Ed25519' && typeof maybeIdentityStorage === 'string') {
```

Comment:
I wonder if strings like `Ed25519` and `ECDSA` would be better off defined as enums or consts.

Reply:
I can move it to a const

Reply:
although, that doesn't play well with the actual parameter